app/presenters/case_mail_presenter.rb

```diff
@@ -0,0 +1,39 @@
+class CaseMailPresenter < SimpleDelegator
+  # Initialise with a TribunalCase instance and any method not
+  # defined in this class will be forwarded automatically to that instance.
+
+  # GOV.UK Notify expects this to be literals
+  def show_case_reference?
+    case_reference.present? ? 'yes' : 'no'
+  end
+
+  # GOV.UK Notify does not accept `nil` values, instead use empty strings
+  def case_reference
+    tribunal_case.case_reference.to_s
+  end
+
+  # Email address of whoever filled the form out first. If it's the representative,
+  # they would get it, otherwise the taxpayer would
+  def recipient_email
```

Comment:
It's just a matter of preference, but personally I'd write this as;

`started_by_representative? ? representative_contact_email : taxpayer_contact_email`

But, I'm not bothered, either way